install.sh

```diff
@@ -1,59 +0,0 @@
-#!/bin/bash
```

Comment:
Can we leave this as an example and say feel free to use it but caveat: this will change your pytorch version or something like that.

Reply:
I dislike having both an install.sh and the installation instructions in README - it's quite likely someone will update one but not the other and lead to future confusion.

Even if we add caveats to it, that doesn't change the potential for confusing users. We should commit to making sure both work or drop one entirely. I'd prefer the latter.

As to which one to keep, I'm actually in favor of keeping install.sh - but I feel less strongly about that.

Reply:
OK then. Let's keep install instructions in README until we have a conda package.